app/server/routes/v1/post/post.js

```diff
@@ -50,9 +50,9 @@ const Community = require("../../models/community.model");
  *                  $ref: '#/components/schemas/Community/properties/title'
  *      responses:
  *        '404':
- *          description: User or Community not found.
+ *          description: User not found. **||** <br>Community not found.
  *        '403':
- *          description: Forbidden. Community can't have more than 3 pinned posts.
+ *          description: Community can't have more than 3 pinned posts. **||** <br>Must be apart of community to post in community.
```

Comment:
Should be 'a part', not 'apart'.

Reply:
Thanks. You can make code suggestions with the + button and then the "Add suggestion" button, and if I approve of the suggestion I can just click a button to commit the change.